### ISSUE TITLE
Banner via XML causes crash

### ISSUE THREAD

**Issue by Maradox** [AppLovin/AppLovin-MAX-SDK-Android#152]
In actvity.onCreate I initialize the sdk:

```
AppLovinSdk.getInstance(this).mediationProvider = "max"
    AppLovinSdk.getInstance(this).settings.setVerboseLogging(BuildConfig.DEBUG)
    AppLovinSdk.getInstance(this).initializeSdk { configuration: AppLovinSdkConfiguration ->
}
```

The banner is placed in the xml-file of a fragment:

```
<com.applovin.mediation.ads.MaxAdView xmlns:maxads="http://schemas.applovin.com/android/1.0"
    android:id="@+id/applovin_add_view"
    android:layout_width="match_parent"
    android:layout_height="@dimen/map_ad_banner_height"
    android:background="@color/map_adview_background"
    maxads:adUnitId="...."
/>
```

When starting the app I reveive an exception during the inflation: `IllegalArgumentException: Max ad view context is not an activity`.

Is the banner via xml only supported when directly inflating with an activity? Creating and adding the banner programmatically works (even with the fragment-context).

**Comment by thomasmso** [comment]
Hi there - we'll investigate the issue and push the fix out in a release later tonight. Thanks!

**Comment by thomasmso** [comment]
We just released v11.1.1 with the fix, thanks once again for the valuable feedback!
